packages/side-nav/src/sass/_module.scss

```diff
@@ -103,6 +103,10 @@
 .au-side-nav__content {
 	@include AU-space( margin, 0 1unit );
 
+	@include AU-media( sm ) {
```

Comment:
We usually put media queries down the bottom of the section.

Also we have a special breakpoint for the side navigation, see if you can work it out.